pkg/deploy/util/util.go

```diff
@@ -18,10 +18,16 @@ import (
 	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
 	kdeplutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 
+	osapiv1 "github.com/openshift/origin/pkg/api/v1"
 	deployapi "github.com/openshift/origin/pkg/deploy/api"
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
+var (
+	// controllerKind contains the schema.GroupVersionKind for this controller type.
+	ControllerKind = osapiv1.SchemeGroupVersion.WithKind("DeploymentConfig")
```

Comment:
Has to be exported because `util` and `deploymentconfig` are different packages.

It is used in 3 places:
- https://github.com/tnozicka/origin/blob/set-owner-refs-in-rcs-owned-by-dc/pkg/deploy/controller/deploymentconfig/deploymentconfig_controller.go#L111
- https://github.com/tnozicka/origin/blob/set-owner-refs-in-rcs-owned-by-dc/pkg/deploy/util/util.go#L241-L242

and the naming aims to be consistent with [upstream](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L64). I can change it if you wish so?